core/src/js/components/battlegrounds/graph-with-comparison-new.component.ts

```diff
@@ -437,17 +446,20 @@ export class GraphWithComparisonNewComponent {
 	private buildSection(
 		theClass: 'player' | 'average',
 		label: string,
+		turnLabel: string,
+		statLabel: string,
+		deltaLabel: string,
 		delta: number,
 		datapoint: ChartTooltipItem,
 	): string {
 		return `
 			<div class="section ${theClass}">
 				<div class="subtitle">${label}</div>
-				<div class="value">Turn ${datapoint?.label}</div>
-				<div class="value">${datapoint?.value ? 'Stat ' + parseInt(datapoint.value).toFixed(0) : 'No data'}</div>
+				<div class="value">${turnLabel} ${datapoint?.label}</div>
+				<div class="value">${datapoint?.value ? statLabel +' '+ parseInt(datapoint.value).toFixed(0) : 'No data'}</div>
```

Comment:
Should the "no data" be localized here?

Reply:
In fact, I have never seen this inscription in the interface, so I did not consider it necessary to localize it.